scripts/build.sh

```diff
@@ -15,7 +15,7 @@ echo 'Done.'
 echo ''
 
 echo 'Build ES modules...'
-NODE_ENV=es babel src --out-dir es --ignore 'src/**/*/fixtures,src/**/*.spec.js'
```

Comment:
We no longer have `fixtures` directories now that we have migrated to vue-test-utils